docs/reference.md

```diff
@@ -2,6 +2,23 @@
 
 ```yaml
 sylius_resource:
+    mapping:
+        paths: ['%kernel.project_dir%/src/Entity'] # Used for Routes with PHP attributes
+    translation:
+        enabled: true
+        enabled_locales: '%kernel.enabled_locales%'
+        default_locale: '%kernel.default_locale%'
+        locale_provider: 'sylius.translation_locale_provider.immutable'
+    settings:
+        paginate: ~
+        limit: ~
+        allowed_paginate: [10, 20, 30]
+        default_page_size: 10
+        sortable: false
+        sorting: ~
+        filterable: false
+        criteria: ~
+        state_machine_component: winzou # or symfony
```

Comment:
I understand all of these parameters were missing, but it would be more _ordered_ if we add only `enabled_locales` and `default_locale` here and the rest of them in a separate PR 😄 #beingPettyIsMyPassion